_api-reference/cluster-api/cluster-settings.md

```diff
@@ -81,6 +82,9 @@ The following request field parameters are compatible with the cluster API.
 | network.breaker.inflight_requests.limit | String | The limit for the in-flight requests breaker. Default is `100%` of the JVM heap. |
 | network.breaker.inflight_requests.overhead | Integer/Time unit | The constant that all in-flight request estimations are multiplied by to determine a final estimation. Default is `2`. |
 | script.max_compilations_rate | String | The limit for the number of unique dynamic scripts within a defined interval that are allowed to be compiled. Default is 150 every 5 minutes: `150/5m`. |
+| cluster.default.index.refresh_interval | Time unit | Sets the refresh interval what the `index.refresh_interval` setting is not provided. This setting is useful when you want to set a default refresh interval across all indexes in a cluster and also support the `searchIdle` setting. You can only set the interval as low as the `cluster.minimum.index.refresh_interval` setting. |
```

Comment:
The first sentence didn't make sense. Check me on the suggested change. I'm not sure if that's the idea.

I would also break up the two ideas "set a default refresh interval across all indexes in a cluster" and "also support the `searchIdle` setting" into two separate sentences. It can do two things. But, since they seem not at all connected, having them in the same sentence reads awkwardly. Can you also link to a description of `searchidle`? It' not clear how the setting supports it.
```suggestion
| cluster.default.index.refresh_interval | Time unit | Sets the refresh interval when the `index.refresh_interval` setting is not provided. This setting is useful when you want to set a default refresh interval across all indexes in a cluster and also support the `searchIdle` setting. You cannot set the interval lower than the `cluster.minimum.index.refresh_interval` setting. |
```